main.js

```diff
@@ -518,6 +591,40 @@ function initPorts() {
                 }
             }
         }
+
+        // Setup input change handler - only has to be done once no matter how many inputs we have
+        if (haveInputs && gpio) {
+            adapter.log.debug('Register onchange handler');
+            gpio.on('change', (port, value) => {
+                // Ignore buttons as they are handled below
+                if (adapter.config.gpios[port].input == 'in') {
+                    adapter.log.debug('GPIO change on port ' + port + ': ' + value);
+                    if (debounceTimers[port] != null) {
+                        // Timer is running but state changed (must be back) so just cancel timer.
+                        clearTimeout(debounceTimers[port]);
+                        debounceTimers[port] = null;
+                    } else {
+                        // Start a timer and report to state if doesn't revert within given period.
+                        debounceTimers[port] = setTimeout((t_port, t_value) => {
```

Comment:
please make sure to also clear all timeouts in "unload"

Reply:
Done